music21/roman.py

```diff
@@ -1785,6 +1792,11 @@ class RomanNumeral(harmony.Harmony):
     >>> cp(r)
     ['G5', 'B5', 'D6', 'F6']
 
+    >>> r = roman.RomanNumeral('IVd6/5')
+    >>> r.key = key.Key('Eb')
+    >>> cp(r)
+    ['C5', 'E-5', 'G-5', 'A-5']
+
```

Comment:
put this in a test below.  It uses a form that we need to parse but that we're not encouraging.

Reply:
This is in the "OMIT_FROM_DOCS" section so I thought it was an appropriate place for such a thing.

Reply:
AH!!! IT IS!  MY BAD!